tests/validators/base.test.ts

```diff
@@ -0,0 +1,90 @@
+import { s } from '../../src';
+
+describe('BaseValidator', () => {
```

Comment:
Missing tests to verify that all methods and getters return a clone and not the instance itself.

Reply:
Also the tests should use the generic to verify the return types are correct.